src/aldente_server.cpp

```diff
@@ -1,10 +1,68 @@
 #include "aldente_server.h"
 
+#include "util/config.h"
+#include "game/game_state.h"
+#include "physics.h"
+#include "scene_manager.h"
+#include "net/network_manager.h"
 #include "timer.h"
 #include <chrono>
 #include <iostream>
+#include <boost/filesystem.hpp>
+
+
+AldenteServer::~AldenteServer() {
+    GeometryGenerator::destroy();
+    glfwTerminate();
+}
+
+static void glSetup() {
```

Comment:
Okay, so GeometryGenerator needs an OpenGL context in order to generate "vertex buffers" on that machine's GPU (to hold the geometry's vertices). But the server machine doesn't need to draw anything; therefore it shouldn't need to send anything to its own GPU; its GPU should be ice cold.